app/src/main/java/com/quran/labs/androidquran/common/LocalTransationDiplaySort.kt

```diff
@@ -0,0 +1,7 @@
+package com.quran.labs.androidquran.common
+
+class LocalTransationDiplaySort : Comparator<LocalTranslation> {
```

Comment:
Who ate the `s`? :)

Reply:
the 'l' you mean?  yea oops correcting spelling...